source/stylesheets/_variables.scss

```diff
@@ -20,7 +20,7 @@ under the License.
 ////////////////////////////////////////////////////////////////////////////////
 // Use these settings to help adjust the appearance of Slate
 
-@import 'http://cheddargetter.com/app/fonts/TradeGothic.css';
+@import 'http://getcheddar.com/app/fonts/TradeGothic.css';
```

Comment:
I think we might have some cross-domain issues with this. There will likely be a server-side config change necessary. Research required regarding loading fonts across domains.